### PR TITLE
prevent install from failing silently

### PR DESCRIPTION
**Pull request by dylansmith**
Fixes https://github.com/imagemin/jpegtran-bin/issues/87

### CODE REVIEW

File: lib/install.js [hunk 1]
```diff
@@ -28,10 +28,9 @@ bin.run(args, err => {
 			.cmd(cfg)
 			.cmd('make install');
 
-		return builder.run(err => {
-			if (err) {
-				log.error(err.stack);
-				return;
+		return builder.run(buildErr => {
+			if (buildErr) {
+				throw buildErr;
```
Comment (sevaru) [inline lib/install.js]: Maybe it's better to still log error but exits from process with error code?
```js
log.error(err.stack);
process.exit(1);
```

Comment (dylansmith) [reply lib/install.js]: As I understand it throwing the error will result in both of those things happening.

Comment (dylansmith) [reply lib/install.js]: Also, `process.exit(1)` violates `unicorn/no-process-exit` as you can see here: https://travis-ci.org/imagemin/jpegtran-bin/jobs/347744916

Comment (sevaru) [reply lib/install.js]: throwing en error would not logs it properly
``log.error(err.stack);``
Maybe put loging back?

**Comment by dylansmith** [comment]
Very stale, closing

**Pull request closed without merging**